third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -1436,13 +1522,13 @@ fn check_depth_of_type_impl(
 struct Frame {
     pc: u16,
     // Currently being executed function.
-    function: LoadedFunction,
+    function: std::rc::Rc<LoadedFunction>,
     // Locals for this execution context and their instantiated types.
     locals: Locals,
     local_tys: Vec<Type>,
     // Cache of types accessed in this frame, to improve performance when accessing
     // and constructing types.
-    ty_cache: FrameTypeCache,
+    pub(crate) ty_cache: std::rc::Rc<std::cell::RefCell<FrameTypeCache>>,
```

Comment:
I think the convention is not to use `std::...` for types like Rcs, Arcs, in general rust and in the codebase. It is pretty clear which RefCell it is, and makes the code a bit more concise.